Analysis/idsf.tex

```diff
@@ -12,12 +12,19 @@
 The tag-probe mass distributions are then fit to extract \npass\ and \nfail. 
 The fit model is composed of two templates, where one template describes a pure \Zee\ line shape and the other describes the background contributions. 
 The backgrounds to the fits include \wj, diboson, and \ttbar\ productions, which are all negligible and estimated to contribute by less than 1\%. 
-Minor contributions from processes that do not involve true electrons, such as diphoton production with a strongly asymmetric conversion on one of the photons and misidentification of a QCD jet as an electron, are predicted to be negligible from MC studies.
+Minor contributions from processes that do not involve true electrons,
+such as diphoton production with a strongly asymmetric conversion on
+one of the photons and misidentification of a QCD jet as an electron,
+are predicted to be negligible from MC studies. \PH{I am confused,
+  they are negligable yet you fit them. Also 1\% is of the same order as your quoted uncetainties}
```

Comment:
yutaro language. not sure why he said wj ttbar and VV are negligible they are minor but need to be fitted. fixing.